project-tracker backend/migrations/20200618151311_create_actionLabels.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex) {
+    return knex.schema.createTable('actionLabels', (table) => {
+        table.increments();
+        table.string('name');
+        table.timestamp('updatedAt').defaultTo(knex.raw('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'));
```

Comment:
szóval, mint slack-en és kérdeztem, mi ez a model, mihez kapcsolódik és biztos kellenek-e bele a timestamp-ek?

Reply:
A timeRecords táblának a része, egy dropdown menüből lehet kiválasztani, kb. tagként funkcionál. Azért kapott külön táblát (és modelt), mert lehet hozzá felvenni új értékeket, ezért nem lehetett enummal megoldani. A timestampek azért kerültek bele, mert feltételeztem, hogy mivel új elemeket lehet hozzáadni, akkor kellenek oda is.